app/models/grda_warehouse/census/project_type_batch.rb

```diff
@@ -4,6 +4,8 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# NOTE: this has been removed from use around 11/2022
```

Comment:
Can we just delete these? I copied the comment here from other classes which haven't been used since 2018.

Reply:
Yeah, let's remove this if we no longer need it!  Thanks.

Reply:
Ok, I deleted this and others with https://github.com/greenriver/hmis-warehouse/pull/2628/commits/0569c9a6e017594136b6418bd8d690aa15eea383

Should I add migrations to drop tables?

Reply:
let's leave the tables for now.  Maybe a story for table cleanup in a couple of weeks?